### PR TITLE
PrgEnv-pgi support

### PR DESCRIPTION
**Pull request by jgphpc**
PrgEnv-pgi support
* Using ~~https://github.com/hpcugent/easybuild-framework/pull/1342~~ as a reference.
* ( i will close ~~https://github.com/hpcugent/easybuild-framework/pull/1537~~ )

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
@jgphpc please ignore the failing tests for now, there's something wrong on our Jenkins server

cc @JensTimmerman

**Comment by JensTimmerman** [comment]
no lmod on jenkins0 yet, in this very particular place
jenkins test this please

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/toolchains/craypgi.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+CrayPGI toolchain: Cray compilers (PGI) and MPI via Cray compiler drivers (PrgEnv-pgi) minus LibSci minus Cray FFTW
+@author: jgp (CSCS)
```
Comment (boegel) [inline easybuild/toolchains/craypgi.py]: @jgphpc please write out your full name? :)

**Comment by boegel** [comment]
@jgphpc now that #1342 is merged, you should resync this with `develop`

also, the file `easybuild/tools/toolchain/craytoolchain.py` should not be here at all, that's the easyblock that is used to install `Cray*` modules; it lives in the easyblocks repository: https://github.com/hpcugent/easybuild-easyblocks/blob/master/easybuild/easyblocks/generic/craytoolchain.py

I'd love to include this in EasyBuild v2.8.0, but then it should be made ready for merging today or tomorrow...

**Comment by boegel** [comment]
@jgphpc this should be merge ready if you merge https://github.com/jgphpc/easybuild-framework/pull/1 to update this PR...

**Comment by boegel** [comment]
I've retested the HPL easyconfig with the two versions of CrayGNU, CrayIntel and CrayCCE we have included, all working fine.

File: easybuild/toolchains/craypgi.py (hunk 1 above)
Comment (boegel) [inline easybuild/toolchains/craypgi.py]: @jgphpc please fix this?

Comment (jgphpc) [inline easybuild/toolchains/craypgi.py]: @boegel done, do you see anything else wrong ?

Comment (boegel) [inline easybuild/toolchains/craypgi.py]: should be good to go otherwise

**Comment by pforai** [comment]
LGTM, also a fairly small change.

**Comment by boegel** [comment]
Going in, thanks @jgphpc!

**Pull request merged**